### PR TITLE
245 arrows nav versions

### PR DESCRIPTION
**Pull request by yishain11**

### CODE REVIEW

File: frontend/src/components/AddCommnetBox.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import { FC, useContext, useState } from 'react';
```
Comment (nadavmoreno) [inline frontend/src/components/AddCommnetBox.tsx:1]: @yishain11 fix typo in file name

File: frontend/src/components/AddCommnetBox.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+interface AddCommentBoxProps {
+  displayedVersion: Version;
+}
+
+const AddCommentBox: FC<AddCommentBoxProps> = ({ displayedVersion }) => {
+  const { t } = useTranslation();
+  const [newComment, setNewComment] = useState<string>('');
+  const [isSnackbarVisible, setIsSnackbarVisible] = useState(false);
+  const { addComment } = useContext(SectionContext);
+  function handleAddComment() {
```
Comment (nadavmoreno) [inline frontend/src/components/AddCommnetBox.tsx:18]: @yishain11 space up line

File: frontend/src/components/DispalySection.tsx [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+  const { t } = useTranslation();
+  const [isSnackbarVisible, setIsSnackbarVisible] = useState(false);
+  const [currentVersionIndex, setCurrentVersionIndex] = useState<number>(
+    sectionVersions.findIndex((v) => v.id === displayedVersion.id)
+  );
+
+  function getIconColor(voteType: 'up' | 'down'): string {
+    return getVoteColor(theme, voteType, displayedVersion?.my_vote);
+  }
+  function handleShare() {
```
Comment (nadavmoreno) [inline frontend/src/components/DispalySection.tsx:49]: @yishain11 space up line

**Review by nadavmoreno — CHANGES_REQUESTED** [review]

File: frontend/src/components/AddCommnetBox.tsx (hunk 1 above)
Comment (yishain11) [reply frontend/src/components/AddCommnetBox.tsx:1]: done

File: frontend/src/components/AddCommnetBox.tsx (hunk 2 above)
Comment (yishain11) [reply frontend/src/components/AddCommnetBox.tsx:18]: done

File: frontend/src/components/DispalySection.tsx (hunk 3 above)
Comment (yishain11) [reply frontend/src/components/DispalySection.tsx:49]: done

**Pull request merged**